src/RMGPhysics.cc

```diff
@@ -79,7 +79,6 @@ namespace u = CLHEP;
 RMGPhysics::RMGPhysics() {
 
   G4VUserPhysicsList::defaultCutValue = 0.1 * u::mm;
```

Comment:
```suggestion
  G4VUserPhysicsList::defaultCutValue = 0.1 * u::mm;
  this->SetCuts();
```

otherwise SetCuts is not called by default...